pjmedia/include/pjmedia/rtp.h

```diff
@@ -105,11 +105,34 @@ struct pjmedia_rtp_hdr
 };
 #pragma pack()
 
+/**
+ * The first byte of RTP packet additional header.
```

Comment:
As this is a kind of not ordinary header, add a brief desc about the use-case/purpose?